zoo-css-challenge/style.css

```
@@ -66,6 +73,8 @@ button {
   color: #fff;
 }
 
+button:hover {background-color: #cac3e2;}
```

```suggestion
button:focus, button:hover {background-color: #cac3e2;}
```
We should also add `:focus` so when users use the `Tab` key to move to the button, it also changes colour for them